skimage/external/tifffile/tifffile.py

```diff
@@ -70,6 +70,8 @@
 
 Revisions
 ---------
+2018.10.08
+    No longer use numpy.fromstring just use numpy.frombuffer.
```

Comment:
Please, could you indicate what you saif about upstream requires a more recent version of scipy and that this revision is made in our copy to avoid to bump numpy's min version.

Also, is it worth to add something in the TODO list, for when our min numpy version will satisfy the one of tifffile?

I'm OK with this PR, but I just want to make sure that we still understand what we did in 1 year.

Reply:
I think the removal of the vendored code isn't obvious because tifffile is a moving target. It is not something that can just be done blindly because it is very likely that getting to 1.14 will take some time for scikit-image.

I think an issue on github tracking how upstream is dealing with this issue (i.e. to wheels exists for all our platforms) is probably best.